common/src/lib.rs

```diff
@@ -53,3 +53,10 @@ cfg_if::cfg_if! {
         pub mod sentry;
     }
 }
+
+// Time
+cfg_if::cfg_if! {
+    if #[cfg(feature = "std")] {
+        pub mod time;
+    }
+}
```

Comment:
Suggest making this module public in all cases, and only gating the `TimeProvider` implementation on `SystemTime` on the `std` feature.

Reply:
Sounds good, will do and then merge.